content/strategy-goals/strategy/cloud/iam-admin/index.md

```diff
@@ -1,5 +1,47 @@
+
+### Problem Area: Unifying Feature Permissions
+
+Today, nearly all Sourcegraph features (Code insights, batch changes, search notebooks, etc) have unique, custom permissions models, creating confusion for users and preventing cross-feature use cases from being developed.
+
+- Key Result 1: A detailed product & technical plan for implementing a unified permissions model
+- Key Result 2: Implementation of the MVP for a unified permissions model
+- Key Result 3: Robust documentation for feature teams to adopt this updated permissions mode
+
+Within this area, in Q2 we are not working on:
+
+- Implementing RBAC for administrators
+- Updating features to respect our permissions, this will be owned by each feature team
+
+**Product and Engineering OKR Alignment:** Objective #2: Focus on time-to-value for a single use-case (Fixing Vulnerabilities). With a single permissions model, Sourcegraph feature teams will be able to enable cross-feature experiences for our customers to fix vulnerabilities. For example, setting up a code monitor to ensure a vulnerability is not re-introduced after fixing with a batch change.
+
+### Problem Area: Improve Admin Analytics
+
+Today, Sourcegraph Admins do not have a clear understanding of how their team is using (or not using) Sourcegraph. This information is valuable to help administrators (1) convey the value of Sourcegraph to executives and (2) make key decisions required to do their job effectively, such as monitoring remaining licenses.
+
+- Key Result 1: Identify and deliver an improved usage analytics experience for admins.
+
+**Product and Engineering OKR Alignment:** Objective #1: Faster Time-to-Value by enterprises. Today, enterprise admins have very little visibility into usage within their organization. They are forced to rely on the basic stats provided by Sourcegraph or anecdotal reviews from their team. Introducing more fine grained analytics will enable admins to easily quantify value to their team and aid Sourcegraph AE/CE in driving upsell conversations based on usage within the org.
+
+### Problem Area: Allow admins to control sign-ups via externally defined groups
+
+Today, Sourcegraph does not provide administrators the ability to specify who can and cannot sign up for Sourcegraph, yet this functionality is critical for customers who want to adopt a land-and-expand strategy for adoption within their organization.
+
+- Key Result 1: Unblock deployment for blocked customers
+- Key Result 2: A technical roadmap is written to provide admins the ability to limit user sign-up based on an externally defined entity (i.e. a GitHub Organization or Team, Gitlab organization, LDAP group for SAML connections).
+- Key Result 3: Short term improvements are implemented to provide admins the ability to limit user sign up based on externally defined entities (i.e. GitHub organization, GitLab organization, IDAP group)
```

Comment:
Does the order/number of the key results matter? Looking at other key results their order seems making sense to me in the order they are define. But the order here which would make sense to me is 3, 1, 2.

Reply:
@unknwon - The ordering is closer to "How it happened", rather than any amount of importance. The #1 is already done and was *very* hacky solution because these customers were blocked on a production deployment. Then we started to build the plan for #2, and realized there are a few other short term fixes that would make our exiting solution less hacky, while still providing short term value to customers wanting this solution.

 So ultimately, the goal in Q2 is to provide a few short term fixes for customers and gather agreement on a long-term product improvement.

Reply:
Makes sense, thanks!